src/databases/Blueprint/avtBlueprintDataAdaptor.C

```diff
@@ -719,30 +722,38 @@ UnstructuredTopologyToVTKUnstructuredGrid(int domain,
         if (n_topo["elements/shape"].as_string() == "polyhedral" || 
             n_topo["elements/shape"].as_string() == "polygonal")
         {
-            Node s2dmap, d2smap, options;
-            blueprint::mesh::topology::unstructured::generate_sides(
-                n_topo,
-                res["topologies/" + n_topo.name()],
-                res["coordsets/" + n_topo["coordset"].as_string()],
-                s2dmap,
-                d2smap);
-
-            unsigned_int_accessor values = d2smap["values"].value();
-
-            oca = vtkUnsignedIntArray::New();
-            oca->SetName("avtOriginalCellNumbers");
-            oca->SetNumberOfComponents(2);
-
-            for (int i = 0; i < values.number_of_elements(); i ++)
-            {
-                unsigned int ocdata[2] = {static_cast<unsigned int>(domain), 
-                                          static_cast<unsigned int>(values[i])};
-                oca->InsertNextTypedTuple(ocdata);
-            }
+            #if CONDUIT_HAVE_PARTITION_FLATTEN == 1
+                Node s2dmap, d2smap, options;
+                blueprint::mesh::topology::unstructured::generate_sides(
+                    n_topo,
+                    res["topologies/" + n_topo.name()],
+                    res["coordsets/" + n_topo["coordset"].as_string()],
+                    s2dmap,
+                    d2smap);
+
+                unsigned_int_accessor values = d2smap["values"].value();
+
+                oca = vtkUnsignedIntArray::New();
+                oca->SetName("avtOriginalCellNumbers");
+                oca->SetNumberOfComponents(2);
+
+                for (int i = 0; i < values.number_of_elements(); i ++)
+                {
+                    unsigned int ocdata[2] = {static_cast<unsigned int>(domain), 
+                                              static_cast<unsigned int>(values[i])};
+                    oca->InsertNextTypedTuple(ocdata);
+                }
 
-            coords_ptr = res.fetch_ptr(
-                "coordsets/" + n_topo["coordset"].as_string());
-            topo_ptr = res.fetch_ptr("topologies/" + n_topo.name());
+                coords_ptr = res.fetch_ptr(
+                    "coordsets/" + n_topo["coordset"].as_string());
+                topo_ptr = res.fetch_ptr("topologies/" + n_topo.name());
+            #else
+                Node about;
+                conduit::about(about);
+                BP_PLUGIN_EXCEPTION1(InvalidVariableException,
+                    "VisIt Blueprint Plugin requires Conduit >= 0.8.0 to read polytopal meshes."
```

Comment:
It didn't occur to me to ask but I've now encountered the word *polytopal* twice. Is that actually part of the Blueprint data model? I thought this was about arbitrary *polyhedral* meshs.

Reply:
I learned something...didn't know that term before. Thanks